src/cursor.rs

```diff
@@ -25,6 +25,7 @@ pub struct CursorWithOwnership<'l> {
 #[derive(Debug)]
 pub struct Row {
     column_mapping: Rc<HashMap<String, usize>>,
+    column_names: Rc<Vec<String>>,
```

Comment:
Please keep the same order as in `Statement`, that is, `column_names` first.

Reply:
Done